docs/reverse-replication/ReverseReplicationUserGuide.md

```diff
@@ -187,7 +196,7 @@ In this case, check if you observe the following:
 
   1. The source database table does not have a primary key
   2. The primary key value was not present in the change stream data
-  3. Check the shard_skipped_files table created in the metadata database. The contains the intervals for which the file was found in GCS, for the cases when no change record was generated for the interval.If a file is present in the shard_skipped_files table and also exists in GCS - this indicates a data loss scenario - please raise a bug.
+  3. When there is no data written to Spanner for a given interval for a given shard, no file is created in GCS. In such a case, the interval is skipped by the writer Dataflow job. This can be verified in the logs by searching for the text *skipping the file*. If a file is marked as skipped in the logs but it exists in GCS - this indicates a data loss scenario - please raise a bug.
```

Comment:
nit: we should put the skipping the file in code instead of bold? `"skipping the file"`

Reply:
Done